src/site/topic7.rst

```diff
@@ -2,6 +2,18 @@
 Topic #7 --- Links
 ******************
 
+.. warning::
+
+    We are not yet defining a new ADT. We're just prepping to start implementing other ADTs with something other than an array.
+
+
+* Arrays have a fixed size
+    * But we can expand capacity
```

Comment:
Or shrink it! Maybe _expand_ -> _change_.

Reply:
Grand